package.json

```diff
@@ -17,13 +17,19 @@
     "lint-fix:js": "eslint --fix --ext js,vue .",
     "lint-fix:css": "stylelint --fix ./**/*.{less,vue}",
     "test": "npm run test:js && bundlesize",
-    "test:js": "jest --testRegex tests/unit/**/test.*.js"
+    "test:js": "jest --testRegex tests/unit/**/test.*.js",
+    "storybook": "start-storybook -s . -p 6006",
+    "build-storybook": "build-storybook -s ."
```

Comment:
When do I have to run the build step?

Reply:
When I write components for other projects I run `npm run storybook` and it will boot up an instance using the stories defined in the stories folder.

It hot reloads so usually this is the primary way of building coordinates. The assumption being that if it works well in storybook it can be integrated into the site that's using them.

The build-storybook command creates a static folder that can be uploaded somewhere.

Here's one we use for Wikipedia mobile:
https://doc.wikimedia.org/MobileFrontend/master/js/ui/?path=/story/anchor--normal

Happy to demo all this at some point in 2021 if that's helpful.